Src/ILGPU/Backends/OpenCL/CLCodeGenerator.Values.cs

```diff
@@ -256,6 +257,14 @@ public void GenerateCode(IfPredicate predicate)
         /// <summary cref="IBackendCodeGenerator.GenerateCode(GenericAtomic)"/>
         public void GenerateCode(GenericAtomic atomic)
         {
+            if ((atomic.ArithmeticBasicValueType == ArithmeticBasicValueType.Int64 ||
```

Comment:
Suggestion: Maybe we can try to refactor this a bit 🤓

Reply:
Refactored into common method.